rsapi/commands.go

```diff
@@ -104,6 +105,12 @@ func (a *API) ParseCommand(cmd, hrefPrefix string, values ActionCommands) (*Pars
 					name, value)
 			}
 			*coerced = append(*coerced, APIParams{name: val})
+		case "*time.Time":
+			val, err := time.Parse(time.RFC3339, value)
```

Comment:
Does this mean all times that are used by RSC must be RFC3339 type? I know our APIs use various **different** time formats, so how does that work with this?